src/BloomExe/Book/Book.cs

```diff
@@ -1362,6 +1362,21 @@ private IPage GetPageToShowAfterDeletion(IPage page)
 			return _pagesCache[index + 1]; //give the following page
 		}
 
+		public Dictionary<string, IPage> GetTemplatePagesIdDictionary()
```

Comment:
Any easy way we could detect and warn (or at least assert) if book is not a template?